go/def.bzl

```diff
@@ -960,6 +960,22 @@ def cgo_library(name, srcs,
       **kwargs
   )
 
+
+################
+
+_toolchain_map = {
+    'linux': struct(
+        url = "https://storage.googleapis.com/golang/go1.7.linux-amd64.tar.gz",
+        sha256 = ("702ad90f705365227e902b42d91dd1a4" +
```

Comment:
Can you put the whole sha256 on line line? I don't think there is a strict 80 char line limit on .bzl files and the ability to grep by the sha256 without having to remember that it is just the first 32 characters is something I personally enjoy having.